featurebyte/service/offline_store_feature_table_manager.py

```diff
@@ -306,7 +306,13 @@ async def handle_online_disabled_features(
 
         feature_ids_to_remove = await self.feature_service.get_online_disabled_feature_ids()
         feature_table_data = await self.offline_store_feature_table_service.list_documents_as_dict(
-            query_filter={"feature_ids": {"$in": list(feature_ids_to_remove)}},
+            query_filter={
+                "$or": [
+                    {"feature_ids": {"$in": list(feature_ids_to_remove)}},
+                    {"feature_ids": {"$size": 0}},
+                ],
+                "precomputed_lookup_feature_table_info": None,
```

Comment:
Do we want to consider the case when `precomputed_lookup_feature_table_info` does not exist?

Reply:
Good point, will fix.